ACE/ace/Log_Msg.cpp

```diff
@@ -136,7 +138,13 @@ class ACE_Log_Msg_Manager
 ACE_Log_Msg_Backend *ACE_Log_Msg_Manager::log_backend_ = 0;
 ACE_Log_Msg_Backend *ACE_Log_Msg_Manager::custom_backend_ = 0;
 
-u_long ACE_Log_Msg_Manager::log_backend_flags_ = 0;
+#ifdef ACE_ANDROID
+#  define DEFAULT_LOG_BACKEND_FLAGS ACE_Log_Msg::SYSLOG
```

Comment:
ACE_ before and a ifndef check so that the user can set it in his config.h?

Reply:
Yes, I can